src/base.scss

```diff
@@ -0,0 +1,139 @@
+// ported using https://codemirror.net/5/theme/gruvbox-dark.css
+
+div.#{$selector}.CodeMirror {
+	--bg0: #{$base};
+	--bg1: #{$overlay1};
+	--bg4: #{$overlay0};
+	--fg: #{$text};
+	--fg3: #{$subtext0};
+	--gray: #{$subtext1};
+	--blue: #{$blue};
+	--yellow: #{$yellow};
+	--aqua: #{$blue};
+	--orange: #{$peach};
+
+	--primary-bg: #{$base};
+	--current-line: #{$surface0};
+	--selection: #{$surface2};
+	--atom: #{$lavender};
+	--cursor: #{$overlay1};
+	--keyword: #{$red};
```

Comment:
```suggestion
	--orange: #{$peach};

	--primary-bg: #{$base};
	--current-line: #{$surface0};
	--selection: #{$surface2};
	--atom: #{$lavender};
	--cursor: #{$overlay1};
	--keyword: #{$mauve};
```

Reply:
Yeah I was planning to update that as well. We've got all these variables set up for the GitHub userstyle too so we could change or reference from those.

Reply:
1. Maybe we'll leave that for a later PR with more updates?

Reply:
Yeah I just wanted to mention it more than anything